### DISCUSSION TITLE
thank you. you made UE4 editor looks so much better!

### DISCUSSION THREAD

**Discussion by TheMrBigWill** [TheHerobrine/DarkerNodes#89] [General]
TKS
[![f71cefd8a08abe8dc2cc4d9873a793f.png](https://i.postimg.cc/nrRZ2kDD/f71cefd8a08abe8dc2cc4d9873a793f.png)](https://postimg.cc/kRbkXWwJ)

**Comment by hugoattal** [comment]
Ahah, thank you very much 😄 !
